src/components/Card/__stories__/Card.stories.scss

```diff
@@ -0,0 +1,64 @@
+.cards {
```

Comment:
`.card-stories` as a prefix. Use more specific classes to avoid collision in stories. `card`, `panel` are too generic words

Reply:
ec48a2d